src/blackout.c

```diff
@@ -0,0 +1,243 @@
+#include "scrcmd.h"
```

Comment:
Reviewing this file is made difficult by the fact that it was both renamed and substantially modified from `field_black_out.c`.

Reply:
that is somewhat the point